src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.container--blue {
+  background-color: #308dc0;
+  width: 328px;
+  padding: 10px;
+  margin: 20px;
+  margin-bottom: 40px;
+  border: 1px solid red;
+}
+
+.container--yellow {
+  background-color: #ffa500;
+  padding: 10px;
+  border: 1px solid red;
```

Comment:
this properties appears in few places
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.